app/views/admin/SkippedContactsView.coffee

```diff
@@ -126,43 +151,21 @@ SkippedContactsComponent = Vue.extend
     'skipped-contact-info': SkippedContactInfo
     'flat-layout': FlatLayout
   created: co.wrap ->
+    store.registerModule('page', storeModule()) # TODO: Bring back automatic registering/unregistering for RootComponents?
```

Comment:
I'd recommend doing so, presumably with different strings for the type of wrapper, one for page, one for modal, and maybe unique properties for each component, or (I think preferably) just don't support components for this.